src/pages/demo/solar-system/index.md

```diff
@@ -42,7 +35,7 @@ var sunData = document.getElementById('sunImg').src+'',
 	});
 
 	// load sun and center it
-	fabric.Image.fromURL(sunData, function(sunImg) {
+	fabric.Image.fromURL('http://www.fabricjs.com/assets/sun.png', function(sunImg) {
```

Comment:
Should this be?
```suggestion
	fabric.Image.fromURL('https://fabricjs.github.io/assets/sun.png', function(sunImg) {
```

Since all other updates refer to *fabricjs.github.io*.
Altho,  I can see that the path for assets on the current website at *fabricjs.com* is same!

Reply:
i was undecided, we can make all the same later. 
For some reason codepen does not like loading from simple http and current jekyll does not offer https.

Reply:
Ohh I see. So for now maybe we could use *https://fabricjs.github.io domain* everywhere or maybe we culd make it dynamic as follows 😁
```javascript
const host = `${window.location.protocol}//${window.location.hostname}`;
// Or maybe use only https: by default instead of window.location.protocol
// since anyways CodePen doesn't support loading from http: like you mentioned

fabric.Image.fromURL(`${host}/assets/sun.png`, function(sunImg) {
  // do something with the image here
}
```

Reply:
the dynamic would not work from local for codepen.